met-web/src/components/engagement/form/EngagementWidgets/WidgetDrawerContext.tsx

```diff
@@ -106,7 +116,12 @@ export const WidgetDrawerProvider = ({ children }: { children: JSX.Element | JSX
         setWidgetDrawerOpen(open);
     };
 
-    const handleAddContactDrawerOpen = (open: boolean) => {
+    const handleAddContactDrawerOpen = (open: boolean, data?: Contact) => {
+        if (data !== null) {
+            setSelectedContact(data);
+        } else {
+            setSelectedContact(null);
+        }
```

Comment:
no if else, just setSelectedContact(data) if data is null it will set it to null then anyway

Reply:
thank you